osbs/repo_utils.py

```diff
@@ -60,7 +60,9 @@ def __init__(self, dir_path='', file_name=REPO_CONFIG_FILE):
             with open(file_path) as f:
                 self.container = (yaml.load(f) or {})
 
-        modules = self.container.get('compose', {}).get('modules', [])
+        # container values may be set to None
+        container_compose = self.container.get('compose') or {}
+        modules = container_compose.get('modules') or []
```

Comment:
any particular reason for this change?

Reply:
The yaml file can contain empty arguments, which will be set to `None`.

If, for example, get('copose', {}) returns None, the next get (for modules) will fail.